docs/concepts/services-networking/service.md

```diff
@@ -357,12 +357,12 @@ The default is `ClusterIP`.
 `Type` values and their behaviors are:
 
    * `ClusterIP`: Exposes the service on a cluster-internal IP. Choosing this value 
-     makes the service only reachable from within the cluster. This is the 
+     makes the service only reachable from within the pod. This is the 
      default `ServiceType`.
    * `NodePort`: Exposes the service on each Node's IP at a static port (the `NodePort`). 
      A `ClusterIP` service, to which the NodePort service will route, is automatically 
-     created.  You'll be able to contact the `NodePort` service, from outside the cluster, 
-     by requesting `<NodeIP>:<NodePort>`.
+     created.  You'll be able to contact the `NodePort` service, from outside the pod 
+     (but only reachable inside the cluster), by requesting `<NodeIP>:<NodePort>`.
```

Comment:
the original text is correct. your service is already exposed to the internet if your node ip can be reached from outside the cluster, and this is implied.